src/app/content/highlights/components/cardStyles.ts

```diff
@@ -194,7 +194,7 @@ export const mainWrapperStyles = css`
   `}
 
   ${(props: WrapperProps) => props.hasQuery && css`
-    @media (max-width: ${minimalWidthForCards}em) {
+    @media (max-width: ${minimalWidthForCardsWithSearchResults}em) {
       top: 0 !important;
     }
   `}
```

Comment:
don't you need another query here to handle when all sidebars are closed but the screen is too small to show the cards on the sides? the same as above on line 171?9

Reply:
I think the mobile breakpoint is large enough to cover this case. I'll check when servers are back

Reply:
You were right. I've had to add one more breakpoint and fix an existing one